views/styles/chart_styles.js

```diff
@@ -100,4 +100,38 @@ export const chartStyles = (theme) => ({
   publicText: {
     alignSelf: 'center',
   },
+  tooltipContainer: {
+    width: '14vw',
+    padding: '1vw 1vw .5vw 1vw ',
+    background: 'rgba(255, 255, 255, 0.9)',
+  },
+  tooltipHeaderRow: {
+    display: 'flex',
+    flexDirection: 'row',
+    flexWrap: 'wrap',
+    width: '100%',
+    paddingBottom: '2%',
```

Comment:
where are these values coming from? I'm noticing we're using a few different units like %, px, and vw. Can we consolidate to use px or rem?